src/main/scala/dxWDL/compiler/GenerateIRTask.scala

```diff
@@ -106,18 +107,51 @@ case class GenerateIRTask(
   def apply(task: CallableTaskDefinition, taskSourceCode: String): IR.Applet = {
     Utils.trace(verbose.on, s"Compiling task ${task.name}")
 
+    def unwrapParamMeta(
+        paramMeta: Option[MetaValueElement]
+    ): Option[Vector[IR.IOAttr]] = paramMeta match {
+      case None => None
+      case Some(MetaValueElementObject(obj)) => {
+        // Use flatmap to get the "help" and "pattern" keys if they exist
+        Some(obj.flatMap {
+          case ("help", value) => {
```

Comment:
Can consider setting the "help"  and "patterns" as constants in IR.scala since these are used in multiple places